Xamarin.Forms.Controls/Xamarin.Forms.Controls.csproj

```diff
@@ -41,6 +41,15 @@
     <Compile Update="GalleryPages\CollectionViewGalleries\EmptyViewGalleries\EmptyViewLoadSimulateGallery.xaml.cs">
       <DependentUpon>EmptyViewLoadSimulateGallery.xaml</DependentUpon>
       </Compile>
+    <Compile Update="GalleryPages\CollectionViewGalleries\HeaderFooterGalleries\HeaderFooterString.xaml.cs">
```

Comment:
I thought this project was moved to sdk-style and was using wildcards for adding items...